packages/web-service/assets/sass/_general.scss

```diff
@@ -0,0 +1,3 @@
+div.hidden-area {
```

Comment:
Is this necessary? Can't it just be `hidden`?

Reply:
I did not know about <div hidden> I will change

Reply:
`<div hidden>`